scripts/commands/letmego.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- func: letmego
+-- desc: removes most debilitating effects
+---------------------------------------------------------------------------------------------------
+require("scripts/globals/status")
+
+cmdprops =
+{
+    permission = 2,
+    parameters = ""
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!letmego")
+end
+
+function onTrigger(player)
+    player:delStatusEffect(xi.effect.TERROR)
```

Comment:
While this appears to be useful for some things, we don't believe this is necessary to test content. we're already able to del effects on demand and with immortal toggle we can survive anything thrown at us. We can even temporarily become immune to these effects if they are too annoying.

Reply:
I assume you would need to know all the effect IDs to remove these on demand as you're mentioning? And become immune? I found this command to be a very useful all in one if I was testing a mob that had a lot of debuffs (the final fight in A Shantotto Ascension comes to mind, their TP moves apply a lot of debuffs).

Reply:
Nope, don't need to know the IDs since we map a human readable name to them all. just gotta spell them right and not type like ol Teo  ._. ;; There are some blocking effects like poison screen too.

Reply:
I can remove this command from the PR if this isn't wanted in LSB. I just found it to be the same reason for the softgodmode command, it made testing easier. I used this command a lot.

Reply:
discussing with team about getting some easy toggle for debuf immunity in for you instead of mass removal.

Reply:
I think we're going to take this in for now and possibly change the method later, Zach is going to have a look at the  PR before anything is final.

Reply:
I think what we came up with yesterday, was this would make more sense if it went into `!reset`. The big list of effects could also be a table, or, I think we have a binding that does erase or something? Worth looking into.

Reply:
I'm not sure but I think that erasable only covers part of these.  Regardless of what happens here I'd like to go ahead with eventually adding the option to be immune to these during testing. Can't be annoyed by them if they never land amiright?

Reply:
I've removed the !letmego file and added this into !reset with the eraseAllStatusEffect() function. There is still a table to remove effects that aren't able to be erased.